src/js/utils/sorting.js

```diff
@@ -20,8 +20,10 @@ function numSortBy(sortDir, val, nullVal) {
   return sortByDir(sortDir, num);
 }
 
-function numSort(sort, fieldA, fieldB, nullVal) {
-  const sortVal = getSortNum(fieldA, nullVal) > getSortNum(fieldB, nullVal) ? 1 : -1;
+function numSort(sort, a, b, nullVal = 0) {
```

Comment:
hmm..  I dunno about this.  this means that base-0 values would sort the same as `null`.  when in that case you'd set `nullVal` to `-1` or something.. Really the safest default is _likely_ `Number.NEGATIVE_INFINITY`

Reply:
Ah ya good call. Just updated it to use `Number.NEGATIVE_INFINITY` as the default in the last push.

Reply:
TIL about `Number.NEGATIVE_INFINITY` 🤯

Reply:
Same here @shadowhand